Gordon360/Properties/launchSettings.json

```diff
@@ -7,7 +7,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "sqlDebugging": true,
-      "applicationUrl": "https://localhost:51627;http://localhost:51626"
+      "applicationUrl": "https://localhost:4444;http://localhost:4343"
```

Comment:
Please undo this change - updating the application URL should only be done locally, not committed to GitHub. Otherwise, you end up using the same port as someone else every time you pull changes.